sdk/sdk-api/src/main/java/mil/emp3/api/Overlay.java

```diff
@@ -42,41 +42,65 @@ public Overlay(IGeoContainer geoContainer) {
     }
 
     @Override
-    public void addOverlay(IOverlay overlay, boolean visible)
+    public void addOverlay(IOverlay overlay, boolean visible, Object object)
```

Comment:
Yes they are, you will see pairs of each.  You will see one set of four methods, followed by another set of four methods with the same names.  The changes in other classes are similar.